cloudbuild.yaml

```diff
@@ -61,6 +61,7 @@ steps:
   - 'NEXT_PUBLIC_PAGE_LIMIT=${_NEXT_PUBLIC_PAGE_LIMIT}'
   - 'NEXT_PUBLIC_ENCRYPTION_PUBLIC_KEY=${_NEXT_PUBLIC_ENCRYPTION_PUBLIC_KEY}'
   - 'DECRYPTION_PRIVATE_KEY=${_DECRYPTION_PRIVATE_KEY}'
+  - 'SENDGRID_API_KEY=${_SENDGRID_API_KEY}'
```

Comment:
You have put the value in prod trigger right? just a reminder

Reply:
yep, done